### DISCUSSION TITLE
Parser only emits first comment

### DISCUSSION THREAD

**Discussion by jdidion** [pest-parser/pest#799] [Q&A] [answered]
I've written my [grammar](https://github.com/jdidion/pest-wdl/blob/main/src/wdl.pest) with a non-silent `COMMENT` rule because I want to include comments in my AST. However, I find that only the first comment is emitted and all other comments are silent.

**Comment by huacnlee** [comment, marked as answer]
Use `///` like Rust

```
/// This is line doc
foo = { "foo" }

// This will silent
/* This also too */
```

**Comment by jdidion** [comment]
Sorry - my original bug report was referring to the `COMMENT` rule in my own grammar. But it turns out the emitted comments are showing up, just in a different place than I expected them. Feel free to close/delete this issue.
